.github/workflows/ruby_ci.yml

```diff
@@ -12,16 +16,17 @@ jobs:
       - name: Setup Ruby
         uses: ruby/setup-ruby@v1
         with:
-          ruby-version: 2.6
-      - name: Setup
-        run: |
-          gem install bundler
-          bundle install --jobs=3 --retry=3
+          ruby-version: 2.5
```

Comment:
Why the downgrade from 2.6 to 2.5?

Reply:
we can go with 2.7, but 2.6 is actually not used in our environments, as I've recently found out, so we should test on 2.5 or 2.7

Reply:
I'm fine with it, I was just curious what was the intention behind it